toml/README.md

```diff
@@ -0,0 +1,56 @@
+# TOML
+
+This module parse TOML files. It follows as much as possible the [TOML specs](https://github.com/toml-lang/toml). Be sure to read the supported types as not every specs is supported at the moment and the handling in TypeScript side is a bit different.
```

Comment:
Lines should be wrapped at 80 columns. Please run the "format.ts" program on this README